src/harness/scripts/migrate-0.9.x.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
NIT: consider renaming the file to be more descriptive. migrate-values-0.9.x, restructure-values-0.9.x?

Reply:
done